### PR TITLE
Add new loadbalancer customizing docs

### PR DESCRIPTION
**Pull request by theabmitra**

### CODE REVIEW

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: "Customize OCI LoadBalancer IP Address"
```
Comment (michael-cico) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:2]: I think this doc should be more general, e.g., "Customizing Load Balancers in OCI".  One section could be the public/private discussion, another could be how to set the shape, for example.  

I think the section within the document for this discussion should be "Using Private Load Balancers"

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+linkTitle: OCILoadBalancer
+weight: 1
+draft: false
+---
+
+Verrazzano sets up the following Load Balancers on Kuberentes at install time:
+* LoadBalancer for Nginx ingress
+* LoadBalancer for Istio ingress
+
+Verrazzano allows customizing the system LoadBalancers with OCI to provide a combination of public and private ip addresses.
```
Comment (michael-cico) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:13]: You could say here, "Verrazzano allows customizing the LoadBalancers allocated by Oracle Container Engine (OKE) using annotations defined by OKE", then provide a link to those annotations.  

Then have the section "Using Private OCI LoadBalancers", or something like that.

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+---
+title: "Customize OCI LoadBalancer IP Address"
+description: "Customize OCI LoadBalancer IP Address for Verrazzano system and application endpoints"
+linkTitle: OCILoadBalancer
```
Comment (michael-cico) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:4]: Think this should be friendlier, e.g. "OCI Load Balancers", or "Load Balancers in OCI"

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+metadata:
+  name: example-verrazzano
+spec:
+  profile: dev
+  environmentName: default
+  components:
+    ingress:
+      type: LoadBalancer
+      nginxInstallArgs:
+      - name: controller.service.annotations."service\.beta\.kubernetes\.io/oci-load-balancer-shape"
```
Comment (michael-cico) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:40]: dangling `"`

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+  name: example-verrazzano
+spec:
+  profile: dev
+  environmentName: default
+  components:
+    ingress:
+      type: LoadBalancer
+      istioInstallArgs:
+      - name: gateways.istio-ingressgateway.serviceAnnotations."service\.beta\.kubernetes\.io/oci-load-balancer-shape"
+        value: "10Mbps
```
Comment (michael-cico) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:59]: missing end `"`

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
+---
+title: "Customizing Load Balancers on OKE"
+description: "Customizing Load Balancers on OKE for Verrazzano system and application endpoints"
+linkTitle: OKELoadBalancer
```
Comment (michael-cico) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:4]: Make "OKE LoadBalancers", should be human-readable/friendly

Comment (theabmitra) [reply content/en/docs/setup/customizing/ociLoadBalancerIPs.md]: done

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md (hunk 5 above)
Comment (theabmitra) [reply content/en/docs/setup/customizing/ociLoadBalancerIPs.md]: done

**Review by michael-cico — APPROVED** [review]

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+At the time of installation Verrazzano allows users to customize the shape and size of the LoadBalancers created. 
+OCI offers a flexible LoadBalancer which uses Dynamic Shape: 
+* 10 Mbps
+* 100 Mbps 
+* 400 Mbps
+* 8,000 Mbps
+
+More details on service limits and shape can be found [here](https://docs.oracle.com/en-us/iaas/Content/Balance/Tasks/managingloadbalancer.htm#console).
+
+For example, setting up Nginx LoadBalancer with `10Mbps` can be achieved as follows:
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:26]: For example, setting up Nginx LoadBalancer with `10Mbps` can be achieved as follows: -> For example, you can set up an NGINX load balancer with `10Mbps` as follows:

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+  environmentName: default
+  components:
+    ingress:
+      type: LoadBalancer
+      nginxInstallArgs:
+      - name: controller.service.annotations."service\.beta\.kubernetes\.io/oci-load-balancer-shape"
+        value: "10Mbps"   
+```
+
+For example, setting up Istio LoadBalancer with `10Mbps` can be achieved as follows:
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:44]: For example, setting up Istio LoadBalancer with `10Mbps` can be achieved as follows: -> For example, you can set up an Istio load balancer with `10Mbps` as follows:

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+  environmentName: default
+  components:
+    ingress:
+      type: LoadBalancer
+      istioInstallArgs:
+      - name: gateways.istio-ingressgateway.serviceAnnotations."service\.beta\.kubernetes\.io/oci-load-balancer-shape"
+        value: "10Mbps" 
+```
+
+### Using PRIVATE ip addresses with LoadBalancer
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:62]: ip addresses with LoadBalancer -> IP addresses with a load balancer

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+
+At the time of installation Verrazzano allows users to customize the ip address of LoadBalancers created.
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:64]: At the time of installation Verrazzano allows users to customize the ip address of LoadBalancers created. -> At installation, Verrazzano lets you customize the IP address of the load balancers created.

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+
+For example, setting up Nginx to have a PRIVATE LoadBalancer ip while the Istio LoadBalancer is assigned a PUBLIC ip:
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:66]: For example, setting up Nginx to have a PRIVATE LoadBalancer ip while the Istio LoadBalancer is assigned a PUBLIC ip: -> Example of setting up NGINX to have a PRIVATE load balancer IP address while the Istio load balancer is assigned a PUBLIC IP address:

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 12]
```diff
@@ -0,0 +1,123 @@
...
+  environmentName: default
+  components:
+    ingress:
+      type: LoadBalancer
+      nginxInstallArgs:
+      - name: controller.service.annotations."service\.beta\.kubernetes\.io/oci-load-balancer-internal"
+        value: "true"    
+```
+
+Example of setting up Nginx to have a PUBLIC LoadBalancer ip while the Istio LoadBalancer is assigned a PRIVATE ip:
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:84]: Example of setting up Nginx to have a PUBLIC LoadBalancer ip while the Istio LoadBalancer is assigned a PRIVATE ip: -> Example of setting up NGINX to have a PUBLIC load balancer IP address while the Istio load balancer is assigned a PRIVATE IP address:

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+  components:
+    ingress:
+      type: LoadBalancer      
+    istio:
+      istioInstallArgs:
+        - name: gateways.istio-ingressgateway.serviceAnnotations."service\.beta\.kubernetes\.io/oci-load-balancer-internal"
+          value: "true"
+```
+
+Example of setting both Nginx and Istio to have a PRIVATE LoadBalancer ips:
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:103]: Example of setting both Nginx and Istio to have a PRIVATE LoadBalancer ips -> Example of setting both NGINX and Istio to have a PRIVATE load balancer IP address:

**Review by rosemarymarano — COMMENTED** [review]
Lots of edits required. Please let me know when you have incorporated them.

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 14]
```diff
@@ -0,0 +1,123 @@
+---
+title: "Customizing Load Balancers on OKE"
+description: "Customizing Load Balancers on OKE for Verrazzano system and application endpoints"
```
[the remark this answers is not in the document]
Comment (theabmitra) [reply content/en/docs/setup/customizing/ociLoadBalancerIPs.md:3]: ok

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 15]
```diff
@@ -59,11 +59,11 @@ spec:
         value: "10Mbps" 
 ```
 
-### Using PRIVATE ip addresses with LoadBalancer
+### Using PRIVATE IP address with load balancer
```
Comment (rosemarymarano) [inline content/en/docs/setup/customizing/ociLoadBalancerIPs.md:62]: Using PRIVATE IP address with load balancer -> Using a PRIVATE IP address with a load balancer

**Review by rosemarymarano — COMMENTED** [review]
Just two minor fixes, otherwise ok.

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md (hunk 15 above)
Comment (theabmitra) [reply content/en/docs/setup/customizing/ociLoadBalancerIPs.md:62]: done

File: content/en/docs/setup/customizing/ociLoadBalancerIPs.md [hunk 16]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Customizing Load Balancers on OKE"
-description: "Customizing load balancers on OKE for Verrazzano system and application endpoints"
+descrIP addresstion: "Customizing load balancers on OKE for Verrazzano system and application endpoints"
```
[the remark this answers is not in the document]
Comment (theabmitra) [reply content/en/docs/setup/customizing/ociLoadBalancerIPs.md:3]: done

**Review by rosemarymarano — APPROVED** [review]
Done.

**Pull request merged**